torchaudio/transforms.py

```diff
@@ -60,6 +60,20 @@ class Spectrogram(torch.nn.Module):
             dimension for real and imaginary parts. (see ``torch.view_as_real``).
             When ``power`` is provided, the value must be False, as the resulting
             Tensor represents real-valued power.
+
+         Example:-
```

Comment:
1. Indentation is off.
2. `hyphen` and `colon` are unnecessary.

Please follow an existing example;

https://github.com/pytorch/audio/blob/aec0e8c9289920e4971fb0b4bbe5b5942a7103f4/torchaudio/functional/functional.py#L821-L824